SwiftyCollection/Classes/CollectionExtensions.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  CollectionExtensions.swift
+//  Pods
+//
+//  Created by Badreddine El Jamali on 2018-06-04.
+//
+
+import Foundation
+
+
+public extension Collection {
+    public func element(at index: Index) -> Element? {
+        if indexInBounds(index) {
```

Comment:
According to our swift guidelines, I would prefer `guard`

Reply:
done